ConfigurationTool/ConfigToolRuntime.cs

```diff
@@ -43,6 +43,13 @@ public ConfigToolRuntime(IServiceProvider provider, FullConfig config, FullConfi
             this.baseConfig = baseConfig;
             this.output = output;
             log = provider.GetRequiredService<ILogger<UAServerExplorer>>();
+
+            // Set retry config lower here, otherwise this might take a really, really long time.
+            // We do a few retries, just to avoid the worst flakyness on the server.
+            config.Source.Retries = new UARetryConfig
```

Comment:
Would be good to externally document that the configuration tool ignores the set value for retries.

Reply:
Yeah, we should add that to the docs page on the config tool.

Reply:
I think we should also have some indicator in the following places:
- https://github.com/cognitedata/opcua-extractor-net/blob/f1ee3bd8fe26f1bfeb21345cc4b6379301bdb655/Extractor/Config/SourceConfig.cs#L202
- https://github.com/cognitedata/opcua-extractor-net/blob/f1ee3bd8fe26f1bfeb21345cc4b6379301bdb655/config/config.example.yml#L134

Reply:
Sure.